fileio/ft_read_event.m

```diff
@@ -1548,8 +1548,9 @@
         for i=1:hdr.nTrials 
             event(end+1).type      = 'trial';
             event(end  ).sample    = begsample(i);
-            event(end  ).value     = events_label(events_code == hdr.orig.epochs.events(i, 3), :);
+            event(end  ).value     = hdr.orig.epochs.events(i, 3);
             event(end  ).offset    = -hdr.nSamplesPre;
+            event(end  ).label     = events_label(events_code == hdr.orig.epochs.events(i, 3), :); % is not used in further functions
```

Comment:
this breaks the API and documentation of the function, see http://www.fieldtriptoolbox.org/development/fileio. That is bad,  not only for FT but also for other toolboxes that use fileio (like SPM and EEGLAB).

event.value can be either numeric or strings. If all string-values can be interpreted as numeric, then they should be converted to numbers.

I propose something like

```
if ~any(isnan(cellfun(@str2double, {event.value})))
  % they can all be converted to numeric values
  for i=1:numel(event)
    event(i).value = str2double(event(i).value);
 end
end 
```

Reply:
this be after the for loop, and event.label would not be created.

Reply:
Then, I would propose to remove all processing of the event labels. I was looking for a way to get the label from MNE-Python into Fieldtrip. The event information in Python saved as dictionary. Matlab reads it as string vector:

`hdr.orig.epochs.event_id = 'cond1:11;cond2:12;cond3:13;cond4:14'`

In my first attempt, I focussed on the label ignoring the numeric code. But that makes further processing invonvenient because of the missing vector in `data.cfg.trl`.

Probably it is best to focus on the numeric values and the user has to know the corresponding mapping.